src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.cs

```diff
@@ -509,15 +481,24 @@ public override bool Equals(object obj)
                i == program.i &&
                S == program.S;
     }
-}",
-index: 0,
-parameters: CSharp6Implicit);
+}";
+
+            var test = new VerifyCS.Test
+            {
+                TestCode = code,
+                FixedCode = fixedCode,
+                CodeActionIndex = 0,
+                LanguageVersion = LanguageVersion.CSharp6,
+            };
+
+            test.Options.AddRange(PreferImplicitTypeWithInfo());
```

Comment:
can this not be written as:

```c#
            var test = new VerifyCS.Test
            {
                TestCode = code,
                FixedCode = fixedCode,
                CodeActionIndex = 0,
                LanguageVersion = LanguageVersion.CSharp6,
                Options = { PreferImplicitTypeWithInfo() },
            };
```

Reply:
Did I add an `Add` overload for a collection? If so that would be cool 😆

Reply:
Currently it's an `AddRange` so this initializer syntax doesn't work for it:

https://github.com/dotnet/roslyn/blob/4aaaed7a97adee9d1c4b8c016d66b62e77e48704/src/Workspaces/CoreTestUtilities/OptionsCollection.cs#L54

Reply:
perhaps something to consider to help make this nicer :)

Reply:
@jaredpar have you seen this scenario come up in LDM?